src/customers/store-credit.md

```diff
@@ -2,7 +2,7 @@
 ee_only: true
 title: Store Credit
 redirect_from:
-  - /sales/store-credit-workflow.html
+  - /sales/store-credit.html
```

Comment:
Let's leave the existing redirect along with the new one. I will update the contributor guide wiki page to add this use case. The system does support more than one redirect.

```suggestion
  - /sales/store-credit-workflow.html
  - /sales/store-credit.html
```